GameOfRunes/GameOfRunes/Scenes/GameHomeScene.swift

```diff
@@ -457,11 +469,6 @@ extension GameHomeScene {
     }
     
     private func startGameSuccess() {
-        // placeholder to prevent crash
-        gameStateMachine?.avatar = .elementalWizard
-        gameStateMachine?.stage = GameViewController.storage.load(stageName: "Cathedral Mayhem")
-        
-        // @brian - room will contain everything u need for the setting up of the scenes
         gameStateMachine?.room = room
```

Comment:
Should we set `avatar` and `stage` to `nil`? Likewise in single player, set `room` to `nil`?

Reply:
yeah I guess that works also, multiplayer pulls from `room` while single player pulls from `avatar` and `stage`

Reply:
yup can. Done!